packages/hoppscotch-sh-admin/src/pages/users/invited.vue

```diff
@@ -12,69 +12,34 @@
 
     <div class="flex flex-col">
       <div class="py-2 overflow-x-auto">
-        <div>
-          <div v-if="fetching" class="flex justify-center">
-            <HoppSmartSpinner />
-          </div>
-          <div v-else-if="error || invitedUsers === undefined">
-            <p class="text-xl">{{ t('users.no_invite') }}</p>
-          </div>
-
-          <table v-else class="w-full text-left">
-            <thead>
-              <tr
-                class="text-secondary border-b border-dividerDark text-sm text-left"
-              >
-                <th class="px-3 pb-3">{{ t('users.admin_id') }}</th>
-                <th class="px-3 pb-3">{{ t('users.admin_email') }}</th>
-                <th class="px-3 pb-3">{{ t('users.invitee_email') }}</th>
-                <th class="px-3 pb-3">{{ t('users.invited_on') }}</th>
-              </tr>
-            </thead>
-            <tbody class="divide-y divide-divider">
-              <tr
-                v-if="invitedUsers.length === 0"
-                class="text-secondaryDark py-4"
-              >
-                <div class="py-6 px-3">{{ t('users.no_invite') }}</div>
-              </tr>
-              <tr
-                v-else
-                v-for="(user, index) in invitedUsers"
-                :key="index"
-                class="text-secondaryDark hover:bg-zinc-800 hover:cursor-pointer rounded-xl"
-              >
-                <td class="py-2 px-3 max-w-30">
-                  <div>
-                    <span class="truncate">
-                      {{ user?.adminUid }}
-                    </span>
-                  </div>
-                </td>
-                <td class="py-2 px-3">
-                  <span class="flex items-center">
-                    {{ user?.adminEmail }}
-                  </span>
-                </td>
-                <td class="py-2 px-3">
-                  <span>
-                    {{ user?.inviteeEmail }}
-                  </span>
-                </td>
-                <td class="py-2 px-3">
-                  <div class="flex items-center">
-                    <div class="flex flex-col">
-                      {{ getCreatedDate(user?.invitedOn) }}
-                      <div class="text-gray-400 text-xs">
-                        {{ getCreatedTime(user?.invitedOn) }}
-                      </div>
-                    </div>
-                  </div>
-                </td>
-              </tr>
-            </tbody>
-          </table>
+        <div v-if="fetching" class="flex justify-center">
+          <HoppSmartSpinner />
         </div>
+
+        <p v-else-if="error" class="text-xl">
+          {{ t('users.invite_load_list_error') }}
+        </p>
```

Comment:
```suggestion
        <div v-else-if="error" class="text-xl">
          {{ t('users.invite_load_list_error') }}
        </div>
```

Since it's a short message, I think we can keep using `div`; also because it has been the convention followed.